src/Chartjs/assets/src/chart_v4_controller.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+import AbstractChartController from './abstract_controller';
+import Chart from 'chart.js/auto';
+import { ChartConfiguration, ChartItem } from 'chart.js';
+
+export default class extends AbstractChartController {
+    createChart(canvasContext: ChartItem, payload: ChartConfiguration): any {
+        return new Chart(canvasContext, payload);
+    }
+}
```

Comment:
Yes, that's right they do :) Because (as commented above) I was a bit stuck on the importing `chart.js` part, as the documentation states for all versions (`v3.8.2`, `v3.9.1` and `v4.0`) the import is the same (just `import Chart from 'chart.js/auto';`),  but the [tests](https://github.com/symfony/ux/actions/runs/3696243151/jobs/6259682636) in this PR gave an error on the import... 😕 so it was still a WIP :)

Reply:
So the difference in 'importing' should be added specifically to the `v3` and `v4` controller files.